opentech/apply/users/models.py

```diff
@@ -1,5 +1,31 @@
-from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.contrib.auth.models import AbstractUser, BaseUserManager
+from django.utils.translation import gettext_lazy as _
+
+from .utils import send_activation_email
+
+
+def convert_full_name_to_parts(full_name):
+    first_name, last_name = full_name.split(' ', 1)
+    return first_name, last_name
+
+
+class UserManager(BaseUserManager):
+    def get_or_create(self, defaults, **kwargs):
+        defaults.update(is_active=False)
+
+        first_name, last_name = convert_full_name_to_parts(defaults.pop('full_name', ''))
+        defaults.update(first_name=first_name, last_name=last_name)
+
+        user, created = super().get_or_create(defaults=defaults, **kwargs)
+        if created:
```

Comment:
As discussed move this to a `create_and_notify()` method that defers to `get_or_create()`. The activation email is only sent to new users that come from submissions